mlir/lib/Conversion/QCOToJeff/QCOToJeff.cpp

```
@@ -1497,34 +1184,57 @@ struct QCOToJeff final : impl::QCOToJeffBase<QCOToJeff> {
+          JEFF_BASE_ADJOINT);                                                  \
+    } else if constexpr ((JEFF_KIND) == ::mlir::mqt::gates::JeffKind::PPR) {   \
+      patterns.add<ConvertQCOPPRGateToJeff<QCO_OP>>(                           \
+          typeConverter, context, &state, (JEFF_PPR).p0, (JEFF_PPR).p1);       \
+    } else if constexpr ((JEFF_KIND) ==                                        \
+                         ::mlir::mqt::gates::JeffKind::SpecialU2ToU) {         \
+      patterns.add<ConvertQCOU2OpToJeff>(typeConverter, context, &state);      \
+    }                                                                          \
+  } while (false);
+    // NOLINTEND(bugprone-macro-parentheses)
+
+    MQT_GATE_TABLE(MQT_ADD_QCO_TO_JEFF_GATE)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If MQT_GATE_TABLE is a macro then please configure it.

(unknownMacro)